src/SFA.DAS.AANHub.Api.UnitTests/Controllers/CalendarEventsControllerTests/CalendarEventsControllerPutGuestsTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using FluentValidation.Results;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Api.Controllers;
+using SFA.DAS.AANHub.Api.Models;
+using SFA.DAS.AANHub.Application.Common;
+using SFA.DAS.AANHub.Application.EventGuests.PutEventGuests;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Api.UnitTests.Controllers.CalendarEventsControllerTests;
+
+public class CalendarEventsControllerPutGuestsTests
+{
+    [Test, AutoData]
+    public async Task PutEventGuests_InvokesMediator(Guid requestedByMemberId, Guid calendarEventId, PutEventGuestsModel model, CancellationToken cancellationToken)
+    {
+        Mock<IMediator> mediatorMock = new();
```

Comment:
Some of the setup and mocks could be taken into SETUP method to avoid repetition